tests/common/schemas/test_phonons.py

```diff
@@ -58,3 +59,31 @@ def test_phonon_bs_dos_doc():
 def test_model_validate(model_cls):
     validated = model_cls.model_validate_json(json.dumps(model_cls(), cls=MontyEncoder))
     assert isinstance(validated, model_cls)
+
+
+def test_check_phonon_output_filenames(clean_dir, si_structure):
+    from jobflow import run_locally
+
+    from atomate2.forcefields.flows.phonons import PhononMaker
+    from atomate2.forcefields.jobs import CHGNetRelaxMaker, CHGNetStaticMaker
+
+    phojob = PhononMaker(
+        bulk_relax_maker=CHGNetRelaxMaker(steps=5),
+        phonon_displacement_maker=CHGNetStaticMaker(),
+        static_energy_maker=CHGNetStaticMaker(),
+        store_force_constants=False,
+        generate_frequencies_eigenvectors_kwargs={
+            "filename_BS": "phonon_BS_test.png",
+            "filename_DOS": "phonon_DOS_test.pdf",
+        },
+    ).make(structure=si_structure)
```

Comment:
can we avoid running the workflow given we're only testing the `from_forces_born` method?

Reply:
I have moved the tests now to `tests/forcefields/flows/test_phonon.py`.  It's not a schemas test anymore, then, but that way, we can avoid unnecessary run time.